### PR TITLE
Use different partial for worksheet and guided view

### PR DESCRIPTION
**Pull request by allenwq**
Depends on #700

### CODE REVIEW

**Comment by lowjoel** [comment]
Rebase please

File: app/views/course/assessment/submissions/_worksheet.html.slim [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              th = @submission.class.human_attribute_name(:submitted_at)
+              td = format_datetime(@submission.submitted_at)
+            tr
+              th = @submission.class.human_attribute_name(:grader)
+              td = @submission.graders.uniq
+            tr
+              th = @submission.class.human_attribute_name(:graded_at)
+              td = format_datetime(@submission.graded_at) if @submission.graded_at
+
+        - if @submission.answers.length > 1
```
Comment (allenwq) [inline app/views/course/assessment/submissions/_worksheet.html.slim]: @lowjoel I think we don't want to display the grading summary at the each step of training ?  Can I leave it for now ?

Comment (lowjoel) [inline app/views/course/assessment/submissions/_worksheet.html.slim]: Then, where in the guided submission do you display statistics? Or, are you saying grading summary for guided submission is meaningless?

Comment (allenwq) [inline app/views/course/assessment/submissions/_worksheet.html.slim]: yeah, at least we should not display them in each step. If we want it we should display the summary and all the answers in a separate page after the guided submission was submitted ?  

In any case if this summary can be reused, I'll extract it out into a partial. but for now I can't think a good place to put it.

Comment (lowjoel) [inline app/views/course/assessment/submissions/_worksheet.html.slim]: I don’t know where to place it for trainings either. Factor it out to a partial first, then.

**Comment by allenwq** [comment]
@lowjoel I've factored the grading statistics partial out.

**Pull request merged**